bin/config.toml

```diff
@@ -156,7 +156,9 @@ hostname = "lolcatho.st"
 #path_begin = "/api" # optional
 
 # list of backend servers for this application
-backends = [ "127.0.0.1:1026" ] # list of IP/port
+[[applications.MyApp.backends]]
+address = "127.0.0.1:1026"
+weight = 100
```

Comment:
please document what the weight is, and indicate that 100 is the default if we don't specify it